lib/models/addon.js

```diff
@@ -103,24 +107,43 @@ Addon.prototype.hintingEnabled = function() {
   @private
   @method _requireBuildPackages
  */
+
 Addon.prototype._requireBuildPackages = function() {
```

Comment:
been running this on several apps at work, and working hard to make nice deprecation output (and catching other cases i didn't think of)

I think i got them all now